python/utils/terra_util.py

```diff
@@ -159,6 +161,14 @@ def get_workspace_entity_info(self, use_cache: bool = True) -> dict:
         response = self.request_util.run_request(uri=url, method=GET)
         return json.loads(response.text)
 
+    def get_workspace_acl(self) -> dict:
+        url = f"{self.TERRA_LINK}/workspaces/{self.billing_project}/{self.workspace_name}/acl"
+        response = self.request_util.run_request(
+            uri=url,
+            method=GET
+        )
+        return response.json()
```

Comment:
We're not checking the response code anywhere, which means if this call fails, the downstream code that uses this is going to fail. Maybe we should check the response code, and only return the response.json() if the response code is a valid one. The class that uses this will have to check and make sure there is a response returned then before trying to access anything within the json

Reply:
The response code gets checked in the [request util here](https://github.com/broadinstitute/ops-terra-utils/blob/main/python/utils/request_util.py#L72-L74)